x-pack/plugins/uptime/public/components/functional/monitor_list.tsx

```diff
@@ -79,16 +80,20 @@ export const MonitorList = ({ dangerColor, loading, monitors, primaryColor }: Mo
           },
           {
             field: 'ping.monitor.id',
-            name: i18n.translate('xpack.uptime.monitorList.idColumnLabel', {
-              defaultMessage: 'ID',
+            name: i18n.translate('xpack.uptime.monitorList.nameIdColumnLabel', {
+              defaultMessage: 'Name / ID',
             }),
-            render: (id: string, monitor: LatestMonitor) => (
-              <Link to={`/monitor/${id}`}>
-                {monitor.ping && monitor.ping.monitor && monitor.ping.monitor.name
-                  ? monitor.ping.monitor.name
-                  : id}
-              </Link>
-            ),
+            render: (id: string, monitor: LatestMonitor) => {
+              const name = get(monitor, 'ping.monitor.name');
+              return (
+                <Link
+                  to={`/monitor/${id}`}
+                  title={`Name: ${name ? "'" + name + "'" : 'N/A'}, ID: '${id}'`}
```

Comment:
```suggestion
                  title={`Name: ${name ? `"${name}"` : 'N/A'}\nID: "${id}"`}
```

I think it's better to consistently use the template-style string definition.
Additionally, I think it's nicer and easier to read the title if we add a newline. WDYT?